src/rard/research/migrations/0061_order_in_book.py

```diff
@@ -0,0 +1,108 @@
+# Generated by Django 3.2 on 2023-01-23 11:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("research", "0060_add_reference_model"),
+    ]
+
+    def create_unknown_work(apps, schema_editor):
+        Antiquarian = apps.get_model("research", "Antiquarian")
+        antiquarians = Antiquarian.objects.all()
+        for ant in antiquarians:
+            ant.add_work(name="Unknown Work", unknown=True)
+
+    def create_unknown_book(apps, schema_editor):
+        Work = apps.get_model("research", "Work")
+        works = Work.objects.all()
+        for work in works:
+            work.add_book(name="Unknown Book", unknown=True)
+
+    def assign_links_to_unknown(apps, schema_editor):
+        worklinks = ["FragmentLink", "TestimoniumLink", "AppositumFragmentLink"]
+
+        for link_type in worklinks:
+            links = link_type.object.all()
+            for link in links:
+                ant = link.antiquarian
+                if link.work == None:
+                    link.work = ant.works.get(unknown=True)
+                work = link.work
+                if link.book == None:
+                    link.book = work.books.get(unknown=True)
```

Comment:
I think this has to be `work.book_set.` because the book model doesn't specify a related name on the foreign key